src/test/scala/org/apache/spark/sql/delta/ConvertToDeltaSuiteBase.scala

```diff
@@ -1036,4 +1036,19 @@ trait ConvertToDeltaHiveTableTests extends ConvertToDeltaTestUtils with SQLTestU
       }
     }
   }
+
+  test("convert use the base path to parse partition structure") {
```

Comment:
Nit: maybe change the name to reflect the outcome not the implementation, e.g.,  `can convert a partition-like table path `